src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/WindowsRuntime/Windows/UI/ViewManagement/UISettingsRcw.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Windows.Media;
+
+namespace MS.Internal.WindowsRuntime
+{
+    namespace Windows.UI.ViewManagement
+    {
+        /// <summary>
+        /// Contains internal RCWs for invoking the UISettings
+        /// </summary>
+        internal static class UISettingsRCW
+        {
+
+            public static object GetUISettingsInstance()
+            {
+                const string typeName = "Windows.UI.ViewManagement.UISettings";
+                IntPtr hstring = IntPtr.Zero;
+                Marshal.ThrowExceptionForHR(NativeMethods.WindowsCreateString(typeName, typeName.Length, out hstring));
+                try
+                {
+                    Marshal.ThrowExceptionForHR(NativeMethods.RoActivateInstance(hstring, out object instance));
+                    return instance;
+                }
+                finally
+                {
+                    Marshal.ThrowExceptionForHR(NativeMethods.WindowsDeleteString(hstring));
+                }
+            }
+
+            [Guid("03021BE4-5254-4781-8194-5168F7D06D7B"), InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+            [ComImport]
+            internal interface IUISettings3
+            {
+                [MethodImpl(MethodImplOptions.InternalCall)]
+                void GetIids(out uint iidCount, [MarshalAs(UnmanagedType.LPStruct)] out Guid iids);
+
+                [MethodImpl(MethodImplOptions.InternalCall)]
+                void GetRuntimeClassName([MarshalAs(UnmanagedType.BStr)] out string className);
+
+                [MethodImpl(MethodImplOptions.InternalCall)]
+                void GetTrustLevel(out TrustLevel TrustLevel);
+
+                [MethodImpl(MethodImplOptions.InternalCall)]
+                UIColor GetColorValue([In] UIColorType desiredColor);
```

Comment:
Good thinking, but this is [In] contractually by C# syntax, you don't have to specify it.

Reply:
Done